deploy/provision.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Copyright (c) 2015 Intracom S.A. Telecom Solutions. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v1.0 which accompanies this distribution,
+# and is available at http://www.eclipse.org/legal/epl-v10.html
+# ------------------------------------------------------------------------------
+# Display distribution release version
+#-------------------------------------------------------------------------------
+
+BASE_DIR=$(pwd)
+cd $BASE_DIR
```

Comment:
Yes it is the reference directory in order to use relative paths and avoid full paths. Full paths is not a good practice because it could lead to a distruction
e.g

If you use a full path in case of rm -rf command then if by mistake you miss a space charachter after the first /

rm -rf / home//nstat/controller

accidentally you could erase your root directory